### PR TITLE
Makes the `isorgan()` define more sane

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Changes the `isorgan()` define to `is_external_organ()`
Changes the `is_int_organ()` proc to a define `is_internal_organ()`
Adds 1 new define: `is_organ()`

## Why It's Good For The Game
I just spend 10 minutes debugging why `isorgan()` wouldn't say a fleshy mass was an organ, it's not an intuitive name.

## Testing
It compiled, and I checked the diff twice

## Changelog
NPFC

### CODE REVIEW

File: code/__DEFINES/mob_defines.dm [hunk 1]
```diff
@@ -276,7 +276,9 @@
 
 #define isnewplayer(A)  (istype((A), /mob/new_player))
 
-#define isorgan(A)		(istype((A), /obj/item/organ/external))
+#define is_external_organ(A)		(istype((A), /obj/item/organ/external))
+#define is_internal_organ(A)		(istype((A), /obj/item/organ/internal))
+#define	isorgan(A)				(istype((A), /obj/item/organ))
```
Comment (Contrabang) [inline code/__DEFINES/mob_defines.dm:281]:
```suggestion
#define	is_organ(A)				(istype((A), /obj/item/organ))
```

**Review by Contrabang — COMMENTED** [review]
for consistency?

File: code/__DEFINES/mob_defines.dm (hunk 1 above)
Comment (DGamerL) [reply code/__DEFINES/mob_defines.dm:281]: Forgot that one, good catch! I'll adress this one in VSC

**Review by Contrabang — APPROVED** [review]

**Review by Fordoxia — APPROVED** [review]
Naming things what they actually are is very cool and epic.

**Pull request merged**